examples/classic_controllers/classic_controllers_synch_motor_example.py

```diff
@@ -18,9 +18,9 @@
                         'Finite'    Discrete Action Space
     """
 
-    motor_type = 'PMSM'
-    control_type = 'TC'
-    action_type = 'AbcCont'
+    motor_type = 'SynRM'
+    control_type = 'SC'
+    action_type = 'Finite'
```

Comment:
Nothing against mixing up the example scenarios a bit, but please keep an eye on the balance.